src/client/hud.cpp

```diff
@@ -415,8 +447,13 @@ void Hud::drawStatbar(v2s32 pos, u16 corner, u16 drawdir, const std::string &tex
 	core::dimension2di dstd;
 	if (size == v2s32()) {
 		dstd = srcd;
+#ifdef __ANDROID__ // Android and Desktop
```

Comment:
Incorrect comment. Android only.

Reply:
I left it for the time of WIP!
This should potentially be used on all platforms.